### DISCUSSION TITLE
Clickable commands

### DISCUSSION THREAD

**Discussion by Robinwhood** [m4tt72/terminal#35] [Ideas]
Hello !
Great terminal ! I was trying to add clickable commands to the terminal. 

Instead of just typing 'help', user could also click on help link so that the help command would write in the terminal.

Is there any way I could do that with the project?

**Comment by m4tt72** [comment]
hello @Robinwhood, great suggestion! I'll add that feature to my backlog, and implement it as soon as I can 🙏🏻
